Project/Source/Components/ComponentAudioSource.cpp

```diff
@@ -156,6 +159,12 @@ void ComponentAudioSource::OnEditorUpdate() {
 
 	ImGui::Separator();
 
+	ImGui::DragFloat("Roll Off Factor", &rollOffFact, 1.f, 0.0f, inf);

```

Comment:
Same from ComponentListener. In this case keep the functions on UpdateSourceParameters()